<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>VectorDialogWidget</class>
  <widget class="QWidget" name="VectorDialogWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>535</width>
        <height>432</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>0</number>
      </property>
      <item row="0" column="0" >
        <widget class="QGroupBox" name="sourceGroup" >
          <property name="title" >
            <string>Source</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="_readFromSource" >
                <property name="text" >
                  <string>Read from data source</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <spacer name="spacer3" >
                <property name="sizeHint" >
                  <size>
                    <width>110</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item row="0" column="1" >
              <widget class="QRadioButton" name="_generateX" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>&amp;Generate</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>Generate a new vector with the properties given here as the independent variable.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="KstDataRange" name="_kstDataRange" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="Q3GroupBox" name="_rvectorGroup" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Data Vector Parameters</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="1" column="0" >
              <widget class="QLabel" name="TextLabel6" >
                <property name="text" >
                  <string>Field or column:</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="KUrlRequester" name="FileName" />
            </item>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel4_3" >
                <property name="text" >
                  <string>File name:</string>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1" >
              <widget class="KstComboBox" name="Field" />
            </item>
            <item rowspan="1" row="2" column="0" colspan="2" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <spacer name="spacer2" >
                    <property name="sizeHint" >
                      <size>
                        <width>181</width>
                        <height>21</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QPushButton" name="_connect" >
                    <property name="text" >
                      <string>Conn&amp;ect</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="_configure" >
                    <property name="text" >
                      <string>Con&amp;figure...</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="Q3GroupBox" name="_svectorGroup" >
          <property name="enabled" >
            <bool>false</bool>
          </property>
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Vector values</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel5" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>Number of samples:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>N</cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="1" column="0" colspan="2" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item>
                  <widget class="QLabel" name="TextLabel1_4_2_3" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>From:</string>
                    </property>
                    <property name="buddy" stdset="0" >
                      <cstring>XMin</cstring>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="_xMin" >
                    <property name="text" >
                      <string>-10.0</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>The minimum value of the independent variable.</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="TextLabel4_5_3" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>To:</string>
                    </property>
                    <property name="buddy" stdset="0" >
                      <cstring>XMax</cstring>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="_xMax" >
                    <property name="text" >
                      <string>10.0</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>The maximum value of the independent variable.</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="0" column="1" >
              <widget class="QSpinBox" name="_N" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="maximum" >
                  <number>1000000000</number>
                </property>
                <property name="singleStep" >
                  <number>1000</number>
                </property>
                <property name="value" >
                  <number>1000</number>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The number of samples for the new independent variable.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>KstComboBox</class>
      <extends>QComboBox</extends>
      <header>kstcombobox.h</header>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>_readFromSource</tabstop>
    <tabstop>_generateX</tabstop>
    <tabstop>FileName</tabstop>
    <tabstop>Field</tabstop>
    <tabstop>_configure</tabstop>
    <tabstop>_kstDataRange</tabstop>
    <tabstop>_N</tabstop>
    <tabstop>_xMin</tabstop>
    <tabstop>_xMax</tabstop>
  </tabstops>
  <includes>
    <include location="local" >kstdatarange.h</include>
  </includes>
</ui>
